app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -449,7 +449,8 @@ private fun UserReviewsSection(
                       },
                       options = mapOf(),
                       userViewModel = userViewModel,
-                      reviewViewModel = reviewViewModel)
+                      reviewViewModel = reviewViewModel,
+                      ownerReputationScore = null)
```

Comment:
Why put the ownerReputationScore as null and not simply 0 here?

Reply:
this was a suggestion from Maria, because in the ViewProfileScreen we dont see the owner of the review but the name of the parking being reviewed. So I call the same previewcard but with this argument as null to indicate to the function PreviewCard that it's a call from ViewProfileScreen and that i dont need to see the owner's name with his tag and stuff...

Reply:
Ok, fair enough!